jujugui/static/gui/src/app/components/search-results/search-results.js

```diff
@@ -138,7 +138,8 @@ YUI.add('search-results', function(Y) {
             storeId = obj.storeId;
         if (!storeId || storeId.indexOf(ownerPath) < 0) {
```

Comment:
There is a lot of logic here that makes no sense to the casual reader. Could you add a code comment here outlining why we have to do these things.